data/org.gnome.software.gschema.xml

```diff
@@ -15,11 +15,6 @@
       <summary>Whether to automatically perform updates</summary>
       <description>If enabled, GNOME Software automatically downloads and installs updates in the background (except for some app types like packages, where it prompts the user to install them when ready).</description>
     </key>
-    <key name="refresh-when-metered" type="b">
```

Comment:
Like I commented in #356, it feels wrong to have a downstream patch that removes the setting instead of ignoring the value that we don't want to happen in the code.